Bump sbt to 1.3.0-RC3

eatkins opened the pull request.

eatkins committed:
Bump sbt to 1.3.0-RC3

typesafe-tools commented:
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=develop&zinc=develop&io=pull/249/head&librarymanagement=develop&util=develop&website=develop&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

eatkins committed:
Don't force sbt or scala version on appveyor

We should test the sbt version specified in project/build.properties and
the default scala version.
